packages/beacon-node/src/chain/options.ts

```diff
@@ -89,6 +89,7 @@ export const defaultChainOptions: IChainOptions = {
   blsVerifyAllMultiThread: false,
   disableBlsBatchVerify: false,
   proposerBoostEnabled: true,
+  proposerBoostReorgEnabled: true,
```

Comment:
I'm not sure how much time we need to test this function since we need to be a block proposal. It could be better not to turn this on by default, test in some nodes on holesky/mainnet and turn on later by default when we have enough data. That would help speed up the time to merge this PR